charts/allure/templates/service-api.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "allure.fullname" . }}-api
+  labels:
+    {{- include "allure-api.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
```

Comment:
If you allow setting Service type, it's possible using "LoadBalancer". In that case, can you allow also setting any annotation from values so we can set a DNS name for example?